pkg/gardenlet/controller/shoot/shoot_control_reconcile.go

```diff
@@ -443,6 +443,11 @@ func (r *shootReconciler) runReconcileShootFlow(ctx context.Context, o *operatio
 			Fn:           flow.TaskFn(botanist.DeployVPNShoot).RetryUntilTimeout(defaultInterval, defaultTimeout).SkipIf(o.Shoot.HibernationEnabled),
 			Dependencies: flow.NewTaskIDs(deployGardenerResourceManager, deployKubeScheduler, deployVpnSeedServer, waitUntilKubeAPIServerIsReady),
 		})
+		_ = g.Add(flow.Task{
+			Name:         "Deploying NodeProblemDetector system component",
```

Comment:
```suggestion
			Name:         "Deploying node-problem-detector system component",
```

Reply:
I followed the same convention as [CoreDns](https://github.com/gardener/gardener/blob/88e3fbf7aa948c5ba6101336194206daa41da45d/pkg/gardenlet/controller/shoot/shoot_control_reconcile.go#L419).